_sass/blocks/_revenues_subpage_intro.scss

```diff
@@ -3,32 +3,58 @@
 // This is the first block on a revenues subpage
 //
 // Styleguide blocks.revenues_subpage_intro
+.revenues_subpage-intro_wrapper {
```

Comment:
Selector `revenues_subpage-intro_wrapper` should be written in lowercase with hyphens